cerbero/config.py

```diff
@@ -71,6 +72,17 @@ def __getattr__(self, name):
         except Exception:
             raise AttributeError("%s is not a known variant" % name)
 
+    def __iter__(self):
+        for attr, value in self.__dict__.iteritems():
```

Comment:
Isn't this iterating over all fields, even the private fields __disabled_variants and __enabled_variants ?